### DISCUSSION TITLE
Cannot type in a specific channel from only chatterino

### DISCUSSION THREAD

**Discussion by AlphaSalami** [Chatterino/chatterino2#4963] [Q&A]
I got ban unbanned in a channel. I am not shadowbanned from the channel as I can type from native chat and others can interact. I can type in every other split in my chatterino, except that specific one. I have updated to latest nightly release, cleared cache, tried being banned and unbanned again but none of it works. Is something saved locally that makes chatterino think I'm banned so won't send messages or something? Any help would be appreciated. Thanks

**Comment by pajlada** [comment]
Can you open up a second web chat window & confirm that you messages are coming through when you type in the channel in from web chat?

**Comment by AlphaSalami** [comment]
![image](https://github.com/Chatterino/chatterino2/assets/34898987/74eb7a1e-feb8-4656-b396-b003ee3d02d8)

Message appears in the popout chat I wrote in, in the native chat, and in the chatterino split.

**Reply by pajlada** [reply]
Weird - we don't do anything special from Chatterino. Just connect to IRC & send the message. There's unfortunately nothing we can do about it but can keep the discussion open to see if anyone else has the same issue.

**Reply by AlphaSalami** [reply]
That's okay. I'll update here if I find a solution. Thanks!

**Reply by Felanbird** [reply]
Probably resolved if you get timedout for any duration, this used to happen in rare unban situations but a timeout resolved that issue afaik.

**Reply by AlphaSalami** [reply]
Getting timed out did not fix it. It was fixed for a while after getting banned and unbanned again, however, I just got a disconnect and now the same issue occurred. I don't think it's specific to being banned. It's probably an issue with connecting to the IRC channel in certain situations. Very weird indeed...

**Comment by Felanbird** [comment]
The user who reported this issue likely had some specific shadowban, they were seen typing in the problem chat later that day.
